### PR TITLE
Refactor smoketest into testcase

### PR DESCRIPTION
**Pull request by kislyuk**

### CODE REVIEW

File: tests/test_smoketest.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    if isinstance(command, str):
+        kwargs["shell"] = True
+    print(GREEN(command))
+    try:
+        return runner(command, **kwargs)
+    except CalledProcessError as e:
+        parser.exit(RED(f'{parser.prog}: Exit status {e.returncode} while running "{command}". Stopping.'))
+
+
+@infra.testmode.standalone
```
Comment (Bento007) [inline tests/test_smoketest.py]: I didn't know you could the decorator works on the whole class. That's handy. Also is this an integration test?

File: tests/test_smoketest.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+                res = run(f"{venv_bin}hca dss put-subscription "
+                          "--callback-url https://example.com/ "
+                          "--es-query '{}' "
+                          f"--replica {replica}",
+                          runner=check_output)
+                sub_id = json.loads(res.decode())["uuid"]
+                run(f"{venv_bin}hca dss get-subscriptions --replica {replica}")
+                run(f"{venv_bin}hca dss delete-subscription --replica {replica} --uuid {sub_id}")
+        finally:
```
Comment (Bento007) [inline tests/test_smoketest.py]: This should be in tearDownClass

File: tests/test_smoketest.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    try:
+        return runner(command, **kwargs)
+    except CalledProcessError as e:
+        parser.exit(RED(f'{parser.prog}: Exit status {e.returncode} while running "{command}". Stopping.'))
+
+
+@infra.testmode.standalone
+class Smoketest(unittest.TestCase):
+    def test_smoketest(self):
+        if os.path.exists("dcp-cli"):
```
Comment (Bento007) [inline tests/test_smoketest.py]: This should be in setUpClass

Comment (kislyuk) [reply tests/test_smoketest.py]: Thanks, done.

File: tests/test_smoketest.py (hunk 2 above)
Comment (kislyuk) [reply tests/test_smoketest.py]: Thanks, done.

File: tests/test_smoketest.py (hunk 1 above)
Comment (kislyuk) [reply tests/test_smoketest.py]: D'oh. Corrected.

**Pull request merged**